modules/platform/login/nuxeo-platform-login-ntlm/src/main/java/org/nuxeo/ecm/platform/ui/web/auth/ntlm/NTLMAuthenticator.java

```diff
@@ -30,34 +32,34 @@
 import javax.servlet.http.HttpServletResponse;
 import javax.servlet.http.HttpSession;
 
-import jcifs.Config;
-import jcifs.UniAddress;
-import jcifs.http.NtlmSsp;
-
-import jcifs.smb.NtlmChallenge;
-import jcifs.smb.NtlmPasswordAuthentication;
-import jcifs.smb.SmbAuthException;
-import jcifs.smb.SmbSession;
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 import org.nuxeo.ecm.platform.api.login.UserIdentificationInfo;
 import org.nuxeo.ecm.platform.ui.web.auth.interfaces.NuxeoAuthenticationPlugin;
 
-import static jcifs.smb.NtStatus.NT_STATUS_ACCESS_VIOLATION;
+import jcifs.smb1.Config;
+import jcifs.smb1.UniAddress;
+import jcifs.smb1.http.NtlmSsp;
+import jcifs.smb1.smb1.NtlmChallenge;
+import jcifs.smb1.smb1.NtlmPasswordAuthentication;
+import jcifs.smb1.smb1.SmbAuthException;
+import jcifs.smb1.smb1.SmbSession;
 
 public class NTLMAuthenticator implements NuxeoAuthenticationPlugin {
 
     private static final String JCIFS_PREFIX = "jcifs.";
 
-    public static final String JCIFS_NETBIOS_CACHE_POLICY = "jcifs.netbios.cachePolicy";
+    private static final String JCIFS_SMB1_PREFIX = "jcifs.smb1.";
```

Comment:
why private?

Reply:
To be consistent with the one above it, and avoid a new `/** @since something */` in code that's basically unused too.